components/automate-ui/src/app/modules/configure/configure.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, OnInit } from '@angular/core';
+import { LayoutFacadeService , Sidebar} from 'app/entities/layout/layout.facade';
+import { ConfigService } from 'app/services/config/config.service';
+@Component({
+  selector: 'app-configure',
+  templateUrl: './configure.component.html',
+  styleUrls: ['./configure.component.scss']
+})
+export class ConfigureComponent implements OnInit {
+
+  notificationVisible = false;
+  deploymentType: string
+
+  constructor(
+    private layoutFacade: LayoutFacadeService,
+    private configService: ConfigService
+  ) { }
+
+  ngOnInit(): void {
+    this.layoutFacade.showSidebar(Sidebar.Settings);
+    this.notificationVisible = false;
+    this.getConfigdetails();
+  }
+  
+
+  getConfigdetails() {
+    this.configService.getConfig().subscribe(
+      config => {
+        if(config.deploymentType != 'SAAS') {
+          this.showNotification();
+        } else {
+          this.hideNotification();
+        }
+      }
+    )
+  }
+
+  
+  hideNotification() {
```

Comment:
Why to extract this function for a one line work that can be done directly. Ideally your function is not reusable for now, so we can directly set `this.notificationVisible = true;` from the place called itself